client/src/app/routes/stardust/TransactionPage.tsx

```diff
@@ -224,95 +224,96 @@ class TransactionPage extends AsyncComponent<RouteComponentProps<TransactionPage
-                                inputs={inputs}
-                                unlocks={unlocks}
-                                outputs={outputs}
-                                header="Payload"
-                            />
-                        </div>
-                    )}
-                <div className="section metadata-section">
-                    <div className="section--header section--header__space-between">
-                        <div className="row middle">
-                            <h2>
-                                Block Metadata
-                            </h2>
-                        </div>
-                    </div>
-                    <div className="section--data">
-                        {!metadata && !metadataError && (
-                            <Spinner />
-                        )}
-                        {metadataError && (
-                            <p className="danger">
-                                Failed to retrieve metadata. {metadataError}
-                            </p>
-                        )}
-                        {metadata && !metadataError && (
-                            <React.Fragment>
-                                <div className="section--data">
-                                    <div className="label">
-                                        Is Solid
-                                    </div>
-                                    <div className="value row middle">
-                                        <span className="margin-r-t">
-                                            {metadata?.isSolid ? "Yes" : "No"}
-                                        </span>
-                                    </div>
-                                </div>
-                                <div className="section--data">
-                                    <div className="label">
-                                        Ledger Inclusion
-                                    </div>
-                                    <div className="value row middle">
-                                        <InclusionState
-                                            state={metadata?.ledgerInclusionState}
-                                        />
-                                    </div>
-                                </div>
-                                {conflictReason && (
+                <TabbedSection
+                    tabsEnum={TRANSACTION_PAGE_TABS}
+                    tabOptions={{
+                        [TRANSACTION_PAGE_TABS.Payload]: {
+                            disabled: !inputs || !unlocks || !outputs || transferTotal === undefined,
+                            isLoading: jobToStatus.get("loadBlock") !== PromiseStatus.DONE,
+                            infoContent: transactionPayloadMessage
+                        },
+                        [TRANSACTION_PAGE_TABS.BlockMetadata]: {
+                            isLoading: !metadata && !metadataError
```

Comment:
Add infoContent for Block metadata. The message should be the same as the message on the Transaction Block page `..stardust/block/metadata.json`